export/exporter.go

```diff
@@ -150,6 +150,20 @@ func (e *exporter) export(info protocol.ToolInfo) error {
 	return nil
 }
 
+func deduplicate(values []string) []string {
```

Comment:
The use of **append** ([on line 123](https://github.com/sourcegraph/lsif-go/pull/16/files#diff-14e51e63d8b1ecbb10fbc9c91d32bef6R123)) seems no longer necessary with use of this new **deduplicate** function, since we could accept two `[]string` then merge them into one.

Reply:
I use append after truncating so we can reuse the same allocated space

Reply:
Use `strs := make([]string, s1, len(s1)+len(s2))` inside this function should be effectively same as has an **append** beforehand.